layers/+tools/transmission/packages.el

```diff
@@ -29,50 +29,94 @@
     :defer t
     :init (progn
             (spacemacs/set-leader-keys "att" 'transmission)
+
             (spacemacs/declare-prefix-for-mode 'transmission-mode "ma" "add")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "mg" "goto")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "ms" "set")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "mt" "toggle")
+            (spacemacs/declare-prefix-for-mode 'transmission-mode "mQ" "Queue")
             (spacemacs/set-leader-keys-for-major-mode 'transmission-mode
-              "gt" 'transmission-files
-              "at" 'transmission-add
-              "sd" 'transmission-set-download
-              "gp" 'transmission-peers
-              "gi" 'transmission-info
-              "sl" 'transmission-set-ratio
+              "D"  'transmission-delete
               "m"  'transmission-move
-              "r"  'transmission-remove
-              "ts" 'transmission-toggle
+              "q"  'transmission-quit
+              "R"  'transmission-remove
+              "v"  'transmission-verify
+              "S"  'tabulated-list-sort
+
+              ;; add
               "aa" 'transmission-trackers-add
+              "at" 'transmission-add
+
+              ;; goto
+              "gi" 'transmission-info
+              "gp" 'transmission-peers
+              "gt" 'transmission-files
```

Comment:
what does `t` in `gt` mean?

Reply:
This binding was also brought over as it was from before so I do not know the reason for it. I can change it to `gf` for `[g]o to [f]iles`